spec/support/utils.rb

```diff
@@ -54,9 +54,11 @@ def self.delete_set(client, namespace, set_name)
 
   def self.client
     @client ||= begin
+      host, port = fetch_host_and_port
```

Comment:
I'm not sure why we need this new logic at all? You can just set `AEROSPIKE_HOSTS` and call `Aerospike::Host.new` without passing a hostname and the client will connect to the seed node(s) specified in the env variable. What is the problem that you are trying to solve?

Reply:
@jhecking  Apparently i couldn't make the client work because i've made a typo while declaring `AEROSPIKE_HOSTS`. I'll push the changes soon.